### PR TITLE
Update index page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Update index page
- Removed deprecated classes 

## QA
e.g. 
- go to https://multipass-run-252.demos.haus/
- Compare page against [copy doc](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit#)

## Issue / Card

Fixes [#5508](https://github.com/canonical-web-and-design/web-squad/issues/5508)

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,94 +1,32 @@
...
 {% block content %}
 <section class="p-strip--suru">
   <div class="row">
     <div class="col-6">
       <h1>Ubuntu VMs on demand for any workstation</h1>
-      <p>Get an instant Ubuntu VM with a single command.  Multipass can launch and run virtual machines and configure them with <a href="https://cloud-init.io/" class="p-link--external p-link--inverted">cloud-init</a> like a public cloud. Prototype your cloud launches locally for free.</p>
+      <p>Get an instant Ubuntu VM with a single command.  Multipass can launch and run virtual machines and configure them with <a href="https://cloud-init.io/" class="p-link--inverted">cloud-init</a> like a public cloud.</p>
+      <p><a href="/install" class="p-button--positive">Install now</a></p>
     </div>
     <div class="col-6 u-hide--small u-align--center u-vertically-center">
```
Comment (petesfrench) [inline templates/index.html:15]:
```suggestion
    <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
```

File: templates/index.html [hunk 2]
```diff
@@ -1,94 +1,32 @@
...
   <div class="row u-equal-height u-vertically-center">
     <div class="col-5 u-align--center u-hide--small">
       <img src="https://assets.ubuntu.com/v1/3328b4a0-Cloud-VMs.svg" style="width: 400px;"/>
     </div>
     <div class="col-7">
-      <h2>Cloud-style VMs on-demand</h2>
+      <h2>Cloud-style VMs at your fingertips</h2>
       <p class="p-heading--4">Spin up cloud instances with a single command</p>
-      <p>Launch instances of Ubuntu and initialise them with cloud-init metadata like AWS, Azure, Google, IBM and Oracle clouds. Simulate your own cloud deployment on your workstation.</p>
+      <p>Launch instances of Ubuntu and <a href="https://ubuntu.com/blog/using-cloud-init-with-multipass">initialise them with cloud-init metadata</a> like AWS, Azure, Google, IBM and Oracle clouds. Simulate your own cloud deployment on your workstation.</p>
```
Comment (petesfrench) [inline templates/index.html:29]:
```suggestion
      <p>Launch instances of Ubuntu and <a href="https://ubuntu.com/blog/using-cloud-init-with-multipass">initialise them with cloud-init metadata</a> in the same way you would on AWS, Azure, Google, IBM and Oracle. Simulate your own cloud deployment on your workstation</p>
```

File: templates/index.html (hunk 1 above)
Comment (petesfrench) [reply templates/index.html:15]: Can you also add `u-hide--medium` to the rest of the image of this page please

File: templates/index.html [hunk 3]
```diff
@@ -97,8 +35,8 @@ <h2>Cloud-style VMs on-demand</h2>
   <div class="row u-equal-height u-vertically-center">
     <div class="col-7">
       <h2>VMs for Windows, macOS and Linux</h2>
```
Comment (petesfrench) [inline templates/index.html:37]:
```suggestion
      <h2>Ubuntu VMs on demand for any workstation</h2>
```

File: templates/index.html [hunk 4]
```diff
@@ -97,8 +35,8 @@ <h2>Cloud-style VMs on-demand</h2>
   <div class="row u-equal-height u-vertically-center">
```
Comment (petesfrench) [inline templates/index.html:35]: I think we could also make this section just a `p-strip` as it doesn't look right how it currently is

**Comment by petesfrench** [comment]
There was a section of code I couldn't comment on so I will attach photos.
Content difference:
image

**Comment by minkyngkm** [comment]
Thanks @petesfrench. Sorry there were many texts that I didn't update earlier. All fixed now! Thanks Pete 🙏🏻

**Comment by petesfrench** [comment]
@minkyngkm No worries, it wasn't your fault, the copydoc wasn't updated correctly. Take another look in the doc as there have been some changes in the 'A curated catalogue of images' section (something highlight, some not)

**Review by petesfrench — DISMISSED** [review]
Comments above ^

**Comment by minkyngkm** [comment]
Thanks @petesfrench. I've updated it :)

File: templates/index.html [hunk 5]
```diff
@@ -132,51 +68,51 @@ <h2>Designed for developer convenience</h2>
...
     </div>
   </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-strip">
   <div class="row u-equal-height u-vertically-center">
-    <div class="col-5 u-align--center u-hide--small">
+    <div class="col-5 u-align--center u-hide--small u-hide--medium">
       <img src="https://assets.ubuntu.com/v1/33d66ec7-Image-catalogue.svg" style="height: 171px; width: 300px;"/>
```
Comment (sowasred2012) [inline templates/index.html:80]:
```suggestion
      <img src="https://assets.ubuntu.com/v1/33d66ec7-Image-catalogue.svg" style="height: 171px; width: 300px;" alt="" />
```

File: templates/index.html [hunk 6]
```diff
@@ -132,51 +68,51 @@ <h2>Designed for developer convenience</h2>
...
     <div class="col-7">
       <h2>Appliances in VMs</h2>
       <p class="p-heading--4">Ubuntu Appliance images to try before you Pi</p>
       <p>Run a Virtual Ubuntu Appliance on your workstation without affecting your existing system. No need for spare hardware, just follow the steps and you’ll be up and running in minutes.</p>
-      <a href="https://ubuntu.com/appliance/vm" class="p-button--neutral p-link--external">Try Ubuntu Appliances</a>
+      <a href="https://ubuntu.com/appliance/vm" class="p-button">Try Ubuntu Appliances</a>
     </div>
-    <div class="col-5 u-align--center u-hide--small">
+    <div class="col-5 u-align--center u-hide--small u-hide--medium">
       <img src="https://assets.ubuntu.com/v1/9c8d314f-what-is-an-appiance-header.svg" style="height: 246px; width: 300px;"/>
```
Comment (sowasred2012) [inline templates/index.html:100]:
```suggestion
      <img src="https://assets.ubuntu.com/v1/9c8d314f-what-is-an-appiance-header.svg" style="height: 246px; width: 300px;" alt="" />
```

File: templates/index.html [hunk 7]
```diff
@@ -132,51 +68,51 @@ <h2>Designed for developer convenience</h2>
...
     </div>
   </div>
 </section>
 
-<section class="p-strip--light is-bordered">
+<section class="p-strip is-bordered">
   <div class="row u-equal-height u-vertically-center">
-    <div class="col-5 u-hide--small u-align--center">
+    <div class="col-5 u-hide--small u-hide--medium u-align--center">
       <img src="https://assets.ubuntu.com/v1/5552afb7-community-support.svg" style="height: 220px; width: 220px;"/>
```
Comment (sowasred2012) [inline templates/index.html:108]:
```suggestion
      <img src="https://assets.ubuntu.com/v1/5552afb7-community-support.svg" style="height: 220px; width: 220px;" alt="" />
```

**Review by sowasred2012 — APPROVED** [review]
Looks good - the comments I've made aren't regarding anything introduced by this PR, but there's some quick a11y wins we can make regarding alt text on a few of the images. All of the images I've commented on are decorative and don't add any info an assistive tech user would miss out on, so empty alt tags are fine here, and will prevent screen readers from reading out the ugly filename to their user.

**Pull request merged**